backend/apps/menu/views.py

```
@@ -0,0 +1,259 @@
+        instance = self.get_object()
+        serializer = self.get_serializer(instance, data=request.data, partial=partial)
+        serializer.is_valid(raise_exception=True)
+        self.perform_update(serializer)
+
+        output_serializer = MealSerializer(instance)
+        return Response(output_serializer.data)
+    
+class SpecialMenuRequestViewSet(viewsets.ModelViewSet):
+    serializer_class = SpecialMenuRequestSerializer
+    queryset = SpecialMenuRequest.objects.all()
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)